### ISSUE TITLE
About how to generate ONNX use pytorch?

### ISSUE THREAD

**Issue by Bruce-WangGF** [cassiebreviu/cpp-onnxruntime-resnet-console-app#4]
Please I want to know your code about the torch2onnx, I use a simple code to transfer the torch to onnx, but the result in cpp is very terrible. the following is my code:
`import torch
import torch.onnx
from torchvision import datasets, models, transforms
import torch.nn as nn
import cv2
import numpy as np
from PIL import Image
import nets

model_ft = nets.resnet.resnet18(num_classes=2)
device = torch.device("cpu:0")
model_ft = model_ft.to(device)

mean = [0.485, 0.456, 0.406]
std =  [0.229, 0.224, 0.225]

inference_transform = transforms.Compose([
    transforms.ToTensor(),
    transforms.Normalize(mean, std)
])

model_ft.load_state_dict(torch.load(r'E:\Pycharm_project\classification-pytorch-main\logs\best_epoch_weights.pth'))
dummy_input = np.ones((224,224),dtype=np.uint8)
dummy_input = Image.fromarray(cv2.cvtColor(dummy_input,cv2.COLOR_BGR2RGB))

img_tensor = inference_transform(dummy_input).unsqueeze_(0)

torch.onnx.export(model_ft,img_tensor,'./resnet18.onnx',opset_version=7)

`
Thank you!
